### PR TITLE
Add nicks and emails from all commits into AUTHORS file

### PR DESCRIPTION
**Pull request by 4144**
### Pull Request Prelude
- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
Add nicks and emails from all commits into AUTHORS file.

With duplicates because for some people exists more than one emails.

### CODE REVIEW

**Comment by Emistry** [comment]
there used to be discussion about it many years ago https://github.com/HerculesWS/Hercules/issues/245
just post here in case you might wanna close the PR too

**Comment by 4144** [comment]
well existing authors file without authors is bad.
then better remove it completly.
but normally in authors file present all authors, but now it contains authors from long past only.

File: AUTHORS [hunk 1]
```diff
@@ -1,3 +1,215 @@
...
+Napster <arokaice@live.com>
+Nihadm89 <nihadm89@hotmail.com>
+Noil <senpainoil@gmail.com>
+Normynator <Norman.Ziebal@web.de>
+ossi0110 <asmodus1@web.de>
+ossi0110 <maikkempkes@alice.de>
+panikon <panikon@zoho.com>
+Paradox924x <paradox924x@gmail.com>
+Patskie <escalona3413@gmail.com>
+Phong Truong <phongtruongit@gmail.com>
```
Comment (jenkijo) [inline AUTHORS:160]: this is my personal email, can you change it to Jenkijo <jenkijo@hrzn.community>?

Comment (4144) [reply AUTHORS:160]: This list is auto generated authors from git commits.

if you want email mapping, add them to ``.mailmap``
git will automatically use only given email if see other your emails.

**Pull request merged**